src/core/qgsstringutils.h

```diff
@@ -252,6 +255,15 @@ class CORE_EXPORT QgsStringUtils
      */
     static QString soundex( const QString &string );
 
+    /**
+     * Tests a \a candidate string to see how likely it is a match for
+     * a specified \a search string. Values are normalized between 0 and 1.
+     * \param candidate candidate string
+     * \param search search term string
+     * \since 3.14
+     */
+    static double fuzzyScore( const QString &candidate, const QString &search );
```

Comment:
Since there's other fuzzy string matching functions in this class, I think we need a more explicit name and documentation for this one. Maybe something like "wordInsensitiveMatchScore"? (that's not great... maybe you've got something better? :laughing: )

Reply:
`qgsFuzzyScore` is fine for me, argumentation below.